src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1896,7 +1896,7 @@ protected override void BeginProcessing()
             else
             {
                 // Working Directory not specified -> Assign Current Path.
-                startInfo.WorkingDirectory = ResolveFilePath(this.SessionState.Path.CurrentFileSystemLocation.Path);
+                startInfo.WorkingDirectory = PathUtils.ResolveFilePath(this.SessionState.Path.CurrentFileSystemLocation.Path, this, true);
```

Comment:
Can you use a named arg here so we know what `true` means?

Reply:
I can make that change, sure.

Reply:
Done